framework_tvb/tvb/config/logger/cluster_handler.py

```diff
@@ -42,44 +42,42 @@
 class ClusterTimedRotatingFileHandler(MemoryHandler):
     """
     This is a custom rotating file handler which computes the name of the file depending on the 
-    execution environment (web node or cluster node)
+    execution environment (web node, cluster node or hpc)
     """
 
     # Name of the log file where code from Web application will be stored
     WEB_LOG_FILE = "web_application.log"
 
     # Name of the file where to write logs from the code executed on cluster nodes
-    CLUSTER_NODES_LOG_FILE = "operations_executions.log"
+    ASYNC_OP_LOG_FILE = "operations_executions.log"
 
     # Size of the buffer which store log entries in memory
     # in number of lines
     BUFFER_CAPACITY = 20
 
-
     def __init__(self, when='h', interval=1, backupCount=0):
         """
         Constructor for logging formatter.
         """
         # Formatting string
         format_str = '%(asctime)s - %(levelname)s'
         if TvbProfile.current.cluster.IN_OPERATION_EXECUTION_PROCESS:
```

Comment:
Could we maybe merge the two if conditions"
`if TvbProfile.current.hpc.IN_OPERATION_EXECUTION_PROCESS or TvbProfile.current.cluster.IN_OPERATION_EXECUTION_PROCESS` ?